packages/clay-alert/src/index.tsx

```diff
@@ -162,7 +162,11 @@ const ClayAlert: React.FunctionComponent<IClayAlertProps> & {
 			role="alert"
 		>
 			<ConditionalContainer>
-				<ClayLayout.ContentRow className="alert-autofit-row">
+				<ClayLayout.ContentRow
+					className={classNames('alert-autofit-row', {
+						'align-items-baseline': variant !== 'feedback',
```

Comment:
Instead of adding it here, I think it's better to add it in the CSS. You can add it in this Sass map https://github.com/liferay/clay/blob/a6e08b9db3828c0344aa12254931779b3fd7fa24/packages/clay-css/src/scss/variables/_alerts.scss#L397-L409.